guides/plugins/plugins/storefront/use-media-thumbnails.md

```diff
@@ -0,0 +1,200 @@
+# working with media and thumbnails
+
+## Overview
+
+In Shopware's Storefront, you can assign media objects to the different entities. To name an example, this is often 
+used for products to show more information with images on the product detail page. This guide should give you a 
+starting point on how to use media and thumbnails in your Storefront plugin.
+
+## Prerequisites
+
+In order to build your own subscriber for your plugin, of course you first need a plugin as base. 
```

Comment:
Der satz passt nicht so recht. wir wollen in diesem artikel ja keinen subscriber bauen, sondern ein storefront plugin...

Reply:
Jo, ist mein Fehler, falsch übernommen.